cmd/list/service.go

```diff
@@ -0,0 +1,95 @@
+package list
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/dream11/odin/internal/service"
+	"github.com/dream11/odin/pkg/constant"
+	"github.com/dream11/odin/pkg/table"
+	serviceproto "github.com/dream11/odin/proto/gen/go/dream11/od/service/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var serviceClient = service.Service{}
+var serviceCmd = &cobra.Command{
+	Use:   "service",
+	Short: "List services",
+	Args: func(cmd *cobra.Command, args []string) error {
+		return cobra.NoArgs(cmd, args)
+	},
+	Long: `List services all if no options provided, type odin list service --help for details`,
+	Run: func(cmd *cobra.Command, args []string) {
+		listService(cmd)
+	},
+}
+
+var version, team, label string
+
+func init() {
+	serviceCmd.Flags().StringVar(&name, "name", "", "name of the service")
```

Comment:
the variable `name` used here is defined in cmd/list/environment.go. let's define a separate variable with a different name (eg. `serviceName`) in this file and use it